Rakefile

```diff
@@ -132,7 +139,8 @@ namespace :examples do
   end
 end
 
+task 'test' => 'doctest'
 task 'doctest:test' => 'prepare-converter'
 task 'doctest:generate' => 'prepare-converter'
 # When no task specified, run test.
-task :default => :doctest
+task :default => :test
```

Comment:
We should probably add `test` to travis' script so that the tests run on PR, etc.

Reply:
Good idea and done!